snuba/datasets/session_events_raw.py

```diff
@@ -0,0 +1,87 @@
+from datetime import timedelta
+from typing import Mapping, Sequence
+
+from snuba.datasets.dataset import TimeSeriesDataset
+from snuba.clickhouse.columns import (
+    ColumnSet,
+    DateTime,
+    LowCardinality,
+    Nullable,
+    String,
+    UInt,
+    Float,
+    UUID,
+)
+from snuba.datasets.schemas.tables import MergeTreeSchema
+from snuba.datasets.dataset_schemas import DatasetSchemas
+from snuba.query.extensions import QueryExtension
+from snuba.query.organization_extension import OrganizationExtension
+from snuba.query.processors.basic_functions import BasicFunctionsProcessor
+from snuba.query.processors.prewhere import PrewhereProcessor
+from snuba.query.query_processor import QueryProcessor
+from snuba.query.timeseries import TimeSeriesExtension
+
+
+class SessionEventsRawDataset(TimeSeriesDataset):
+    def __init__(self) -> None:
+        read_columns = ColumnSet(
+            [
+                ("event_id", UUID()),
+                ("session_id", UUID()),
+                ("distinct_id", UUID()),
+                ("seq", UInt(64)),
+                ("org_id", UInt(64)),
+                ("project_id", UInt(64)),
+                ("retention_days", UInt(16)),
+                ("deleted", UInt(8)),
+                ("sample_rate", Float(32)),
```

Comment:
Is it confirmed we will support sampling during ingestion, that will sample sessions consistently (if a session start is sampled then the session end is sampled as well) and that it may change between sessions ?